packit/fedpkg.py

```diff
@@ -126,6 +126,13 @@ def init_ticket(self, keytab: str = None):
         if not keytab and not self.fas_username:
             logger.info("won't be doing kinit, no credentials provided")
             return
+
+        if not self.fas_username:
+            cmd = ["klist", "-k", keytab]
+            if not utils.run_command_remote(cmd=cmd, fail=False):
+                logger.info("won't clone the repo, invalid Kerberos ticket")
```

Comment:
I am not sure about mentioning the cloning in this method (which is called `init_ticket`).

Reply:
shouldn't we raise an exc here?